src/tests/intg/getsockopt_wrapper.c

```diff
@@ -84,6 +101,9 @@ int getsockopt(int sockfd, int level, int optname,
             cr->uid = 0;
         } else if (is_secrets_socket(sockfd)) {
             cr->uid = fake_secret_peer(cr->uid);
+        } else if (peer_is_private_pam(sockfd)) {
+            cr->uid = fake_secret_peer(cr->uid);
```

Comment:
This is a nitpick, but I don't like reusing `SSSD_INTG_SECRETS_PEER` outside tests for the secrets responder. I think it would be easier to just drop "secrets" from the environment variable and the function name and have just `fake_peer` and `SSSD_INTG_PEER`.

Reply:
Fixed in the latest version. I added a new call for PAM usage. It can be reused e.g. for the secrets use-case as well, but I'd prefer to do this with a different PR to avoid breaking existing tests.